.changeset/silent-turtles-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Remove apoc.coll.sortMulti
```

Comment:
I think this might need a bit of an extension/tweak. The PR title suggests something else than the changeset message here. But yes removing `apoc.coll.sortMulti` is definitely part of this PR.

Reply:
mmm, well, this PR does more changes, but I don't think any of those are really worth mentioning in the changelog